src/charm.py

```diff
@@ -398,6 +436,18 @@ def _pop_peer_data(self, key: str) -> Dict:
         data = peers.data[self.app].pop(key, "")
         return json.loads(data) if data else {}
 
+    def _push_ca_certs(self) -> None:
+        self._container.push(CA_CERT_DIR_PATH / "ca_certificates.crt", self._trusted_certs_bundle, make_dirs=True)
```

Comment:
i think we need to run `update-ca-certificates` and push the results of that

https://github.com/canonical/oathkeeper-operator/blob/main/src/charm.py#L464-L486

i guess we could chain them manually like you are doing  https://github.com/canonical/identity-platform-admin-ui-operator/pull/11/files#diff-b9ed39bbc9c0387bd3e07da31d13373745534a1cd723d3e292c73496b12e307cR449 but that wouldn't include the main trustable ones like google or aws or others so you would probably override that

not a problem if we were sticking with self-signed certificates and ensure we always use only that, but in any other case we'd have a problem

Reply:
I don't think we need to trust all those CAs. The admin UI is not going to make any calls to the external/public services. Even if you have a cert issued by one of the "public" CAs I believe that you would use some sort of integrator charm to manage your certificates. So even in that case you would integrate the admin-ui with that charm to trust the ca chain.